gengetopt.yaml

```diff
@@ -0,0 +1,142 @@
+package:
+  name: gengetopt
+  version: 2.23
+  epoch: 0
+  description: "A tool to write command line option parsing code for C programs"
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - libtool
+      - texinfo
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.gnu.org/gnu/gengetopt/gengetopt-${{package.version}}.tar.xz
+      expected-sha256: b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-static
+    pipeline:
+      - uses: split/static
+    description: gengetopt static
```

Comment:
```suggestion

```

Let's omit this since its empty: https://apk.chaindag.dev/https/apk.cgr.dev/wolfi-presubmit/56d7982b85b02eca28b0964836ecc7ae8e7172a2/aarch64/gengetopt-static-2.23-r0.apk@etag:3ab656fbd218cee861b03c325316e87f/

Reply:
Removed, thanks